contracts/external/proxies/LiquidatorProxyV1ForSoloMargin.sol

```diff
@@ -302,6 +291,51 @@ contract LiquidatorProxyV1ForSoloMargin is
 
     // ============ Helper Functions ============
 
+    /**
+     * Make some basic checks before attempting to liquidate an account.
+     *  - Require that the msg.sender is permissioned to use the liquidator account
+     *  - Require that the liquid account is liquidatable
+     */
+    function checkRequirements(
+        Account.Info memory fromAccount,
+        Account.Info memory liquidAccount
+    )
+        private
+        view
+    {
+        // check credentials for msg.sender
+        Require.that(
```

Comment:
Did anything here actually change...?

Reply:
No I just had to move this out because of the 16-variable stack size lol